fixtures/ios_app_with_dynamic_frameworks_linking_static_frameworks/App/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
What about making the fixture leaning by removing this `Assets.xcassets`?

Reply:
Sure 👍 I created it via `tuist init` but definitely no need to have resources here.